pyproject.toml

```diff
@@ -30,7 +30,7 @@ license = {file = "LICENSE"}
 requires-python = ">=3.8,<4"
 dependencies = [
     "numpy >= 1.21.0",
-    "pandas >= 1.5.0, <2.0.0",
+    "pandas >= 2.0.2",
```

Comment:
Again, we need to figure out the correct versioning 🤔

Reply:
Reverted to previous but uncapped.